CalendarApp/GoogleCalendarService/Manager/EventManager.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Google.Apis.Calendar.v3.Data;
+using GoogleCalendarService.Extensions;
+
+namespace GoogleCalendarService.Manager
+{
+    public class EventManager
+    {        
+        private readonly BookingService _bookingService;
+
+        public EventManager(BookingService bookingService)
+        {
+            _bookingService = bookingService;
+        }
+
+        public List<Event> CheckEvent(string calendarId, Event eventForAdd, bool forcedEvent)
+        {
+            if (forcedEvent)
+                return new List<Event>();
```

Comment:
Maybe it is better to skip this forcedEvent parameter and instead modify logic to avoid calling this check method when forcedEvent is true?
@gregory-hasyn  thoughts?